examples/aws/cloudformation/ent.yaml

```diff
@@ -13,6 +13,8 @@ Parameters:
     Default: t2.micro
     AllowedValues:
     - t2.micro
+    - m5.large
+    - m4.large
```

Comment:
This is the full list of instance sizes we "support" from our marketplace listings:

m3.2xlarge
m3.large
m3.medium
m3.xlarge
m4.2xlarge
m4.4xlarge
m4.10xlarge
m4.16xlarge
m4.large
m4.xlarge
t2.large
t2.medium
t2.micro
t2.small
t2.xlarge

Might be worth adding all of these in as options.